tests/bot/exts/utils/test_snekbox.py

```diff
@@ -29,7 +29,7 @@ async def test_post_job(self):
 
         self.assertEqual(await self.cog.post_job("import random", "3.10"), "return")
         self.bot.http_session.post.assert_called_with(
-            constants.URLs.snekbox_eval_api,
+            constants.URLs.snekbox_310_eval_api,
```

Comment:
I'm not sure this is the intended behavior of the test. If the default version is now 3.11 then it makes more sense to change the tests to use 3.11.